apps/cowswap-frontend/src/modules/swap/helpers/getSwapButtonState.ts

```diff
@@ -122,7 +122,7 @@ export function getSwapButtonState(input: SwapButtonStateParams): SwapButtonStat
   }
 
   if (input.inputError) {
-    if (isAprilFoolsEnabled && input.inputError === 'Select a token') {
+    if (!isAprilFoolsEnabled && input.inputError === 'Select a token') {
```

Comment:
I guess you did it for testing, but this change will make it not work as expected

Reply:
Yes for testing, reverting now :)